test_util/test_installer_ccm.py

```diff
@@ -314,6 +314,7 @@ def establish_host_connectivity():
                 master_list=master_list,
                 agent_list=agent_list,
                 public_agent_list=public_agent_list,
+                platform='aws',
```

Comment:
Here's where platform/provider is being set to aws/onprem for the CCM AWS VPC integration tests.